File: Work/Conditions/time.py
#Ryan R, Timed Greeter

import time
#First instance of time in programming
#print(time.gmtime(0),"\n")

#Current time in second since gmtime
#print(time.time(),"\n")

#Current time were used to seeing
curent =time.time()
now = time.ctime(curent)
print(now)

#How to get just the hour
local_time = time.localtime(curent)
hour = local_time.tm_hour
min = local_time.tm_min
sec = local_time.tm_sec
print(f"{hour}:{min}:{sec}")

File: Work/Conditions/time.c
//Ryan Rowbury, Time Checker
#include <stdio.h>
#include <time.h>

int main(void){
    time_t seconds;

    seconds = time(NULL);
    printf("It has been %d seconds since January 1, 1970", seconds);

    time_t rawtime;
    struct tm * timeinfo;

    time(&rawtime);
    timeinfo = localtime(&rawtime);
    printf("\nCurrent time and date is %s", asctime(timeinfo));
    
    time_t now = time(NULL);
    struct tm *tm_truckt = localtime(&now);
    int hour = tm_truckt->tm_hour;
    printf("%d\n", hour);
    return(0);
}